packages/editor-ui/src/modules/templates.ts

```diff
@@ -0,0 +1,38 @@
+import { getTemplateById } from '@/api/templates-mock';
+import { ActionContext, Module } from 'vuex';
+import {
+	IRootState,
+	IN8nTemplate,
+	ITemplateState,
+} from '../Interface';
+import Vue from 'vue';
+
+const module: Module<ITemplateState, IRootState> = {
+	namespaced: true,
+	state: {
+		template: {},
```

Comment:
in state we should support having multiple templates so that we can easily extend this to support search page... 
so this should be mapping template ids to template objects
```
templates: {[id: string]: ITemplate}
```

Reply:
Updated